helper/strutil/strutil_test.go

```diff
@@ -2,6 +2,19 @@ package strutil
 
 import "testing"
 
+func TestStrutil_EquivalentSlices(t *testing.T) {
+	slice1 := []string{"test1", "test2", "test3"}
+	slice2 := []string{"test1", "test2", "test3"}
```

Comment:
I'd put these out of order between slice1 and slice2, just to test that as well.